src/hepqpr/qallse/qallse_base.py

```diff
@@ -274,31 +274,26 @@ def to_qubo(self, return_stats=False) -> Union[TQubo, Tuple[TQubo, Tuple[int, in
         :param return_stats: if set, also return the number of variables and couplers.
         :return: either the QUBO, or a tuple (QUBO, (n_vars, n_incl_couplers, n_excl_couplers))
         """
+        
         #FIXME investigate TQUBO object
-        Q = [[{} for i in range(len(Volayer.phi_slices))] for j in range(len(Volayer.eta_slices))]
-        #qubo_list = [[{} for i in range(len(Volayer.phi_slices))] for j in range(len(Volayer.eta_slices))]
-        quboslice = [[QuboSlice for i in range(len(Volayer.phi_slices))] for j in range(len(Volayer.eta_slices))]
+        #Q = [[{} for i in range(len(Volayer.phi_slices))] for j in range(len(Volayer.eta_slices))]
+        
+        #container class for slices
+        sliceContainer = SliceContainer()
```

Comment:
Create the slice container instance here. Depending on where you want to use it l;ater in the code, you might want to declare it somewhere else and make it a member of some class.